api/api-doc/column_family.json

```diff
@@ -92,6 +92,14 @@
                      "type":"boolean",
                      "paramType":"query"
                   },
+                  {
+                     "name":"force_purge_tombstones",
+                     "description":"Set to \"true\" to flush the memtables and force tombstone garbage collection to check only the sstables being compacted (false by default). The memtable, commitlog and other uncompacted sstables will not be checked during tombstone garbage collection.",
```

Comment:
```suggestion
                     "description":"Set to \"true\" to flush all memtables and force tombstone garbage collection to check only the sstables being compacted (false by default). The memtable, commitlog and other uncompacted sstables will not be checked during tombstone garbage collection.",
```

Reply:
all memtables are already flushed by major compaction regardless of this flag, so it's misleading to include it as part of this description .

Reply:
Also, I'm not entirely sold about the option name, since one could easily understand it as forcing purging of all tombstones regardless of their garbage collection eligibility.  Maybe `aggressive_purge_tombstones` would be better.

That said, we could add another flag to set the tombstone_gc mode for this compaction so one could purge all tombstone regardless of their gc eligibility (by grace seconds or repair), but that can be done as a follow if we see the need to.

@vladzcloudius / @tzach do you think such an option is needed?

Reply:
> all memtables are already flushed by major compaction regardless of this flag, so it's misleading to include it as part of this description .

I see that in the next patchy we actually do force flushing of all memtables with this flag, so please ignore the above comment.